--- setup.py
+++ setup.py
@@ -5,10 +5,8 @@
 VERSION = biopandas.__version__
 PROJECT_ROOT = dirname(realpath(__file__))
 
-REQUIREMENTS_FILE = join(PROJECT_ROOT, 'requirements.txt')
-
-with open(REQUIREMENTS_FILE) as f:
-    install_reqs = f.read().splitlines()
+install_reqs = ["numpy>=1.16.2",
+                "pandas>=0.24.2"]
 
 install_reqs.append('setuptools')
 
@@ -19,10 +17,7 @@
       author_email='mail@sebastianraschka.com',
       url='https://github.com/rasbt/biopandas',
       packages=find_packages(),
-      package_data={'': ['LICENSE.txt',
-                         'README.md',
-                         'requirements.txt']
-                    },
+      package_data={'': ['README.md']},
       include_package_data=True,
       install_requires=install_reqs,
       license='BSD 3-Clause',
